Makefile.ci

```diff
@@ -133,6 +135,12 @@ ci-geocoq: ci-mathcomp
 ci-simple_io: ci-ext_lib
 ci-quickchick: ci-ext_lib ci-simple_io ci-mathcomp
 ci-itree: ci-ext_lib ci-paco
+ci-itree_io: ci-itree ci-simple_io
+
+ci-parsec: ci-ext_lib ci-ceres
+ci-json: ci-parsec ci-menhir
+ci-async_test: ci-itree_io ci-json ci-quickchick
+ci-http: ci-async_test
```

Comment:
> File "./app/AppMessage.v", line 5, characters 0-39:
Error: Cannot find a physical path bound to logical path
Parser with prefix Parsec.

I'm getting confused. How to make `ci-http` aware of Parsec? I thought `Makefile.ci` did the jobs.

Reply:
ci-parsec isn't in the CI_TARGETS list in the makefile (and it's not detected because there's `ci-parsec: ci-ext_lib ci-ceres`)